src/features/script-load.js

```diff
@@ -10,39 +10,40 @@ systemJSPrototype.register = function (deps, declare) {
   systemRegister.call(this, deps, declare);
 };
 
+systemJSPrototype.createScript = function (url) {
+  const script = document.createElement('script');
+  script.charset = 'utf-8';
+  script.async = true;
+  script.crossOrigin = 'anonymous';
+  script.src = url;
+  return script;
+};
+
+let lastWindowErrorUrl, lastWindowError;
+if (hasDocument)
+  window.addEventListener('error', function (evt) {
+    lastWindowErrorUrl = evt.filename;
```

Comment:
Does IE11 support the `filename` property? MDN indicates that it is unknown - https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent

Reply:
No it doesn't, see my response at https://github.com/systemjs/systemjs/pull/1961#issuecomment-508533368 to when you first implemented this.